oc_cdtapi/OnecAPI.py

```
@@ -0,0 +1,229 @@
+class OnecAPI(HttpAPI):
+    """
+    API client for interacting with the 1C system.
+    """
+    
+    _error = OnecError
+    _env_prefix = "ONEC"
+    
+    headers = {
+        "Accept": "application/json;version=2",
+        "Content-Type": "application/json"
+    }
```

Ruff (0.15.4)

[warning] Mutable default value for class attribute

(RUF012)